Tasks/PowerShellOnTargetMachines2/task.json

```diff
@@ -0,0 +1,201 @@
+{
+    "id": "3B5693D4-5777-4FEE-862A-BD2B7A374C68",
+    "name": "PowerShellOnTargetMachines",
+    "friendlyName": "PowerShell on Target Machines",
+    "description": "Execute PowerShell scripts on remote machine(s)",
```

Comment:
Should we call out in description that this is based on Invoke-Command.

Reply:
yes we should do that.